src/ng1/directives/floatingActionButton/floatingActionButton.html

```diff
@@ -2,41 +2,79 @@
     <!-- Additional buttons to the top -->
     <ul class="unstyled p-l-nil child-btn-set-top top m-b-nil" ng-if="direction == 'top'">
         <li ng-repeat="key in items">
-            <button type="button" class="btn btn-icon btn-circular button-secondary m-b-md" aria-label="{{key.icon}}" ng-click="selectedCollapse(key.event)">
-                <span class="{{ key.icon.indexOf('hp-') === -1 ? 'hpe-icon' : 'hp-icon' }}" ng-class="key.icon" aria-hidden="true"></span>
+
+            <button type="button" 
+                    class="btn btn-icon btn-circular button-secondary m-b-md" 
+                    aria-label="{{ key.icon }}" ng-click="selectedCollapse(key.event)"
```

Comment:
This could be improved to make use of the tooltip text if present, e.g. `aria-label="{{ key.tooltip || key.icon }}"`.